lib/measures/tbd/resources/psi.rb

```diff
@@ -1418,12 +1418,12 @@ def glazingAirFilmRSi(usi = 5.85)
 ##
 # Returns a construction's standard thermal resistance (with air films).
 #
-# @param [OpenStudio::Model::Construction] construction An OS construction
+# @param [OpenStudio::Model::LayeredConstruction] lc An OS layered construction
```

Comment:
@macumber: A handful of changes to accommodate both [LayeredConstruction](https://openstudio-sdk-documentation.s3.amazonaws.com/cpp/OpenStudio-3.3.0-doc/model/html/classopenstudio_1_1model_1_1_layered_construction.html) classes in OpenStudio. Non-layered construction classes shouldn't be derated, yet such occurrences shouldn't be tagged as errors either.